checkstyle.xml

```diff
@@ -109,6 +109,9 @@
 		<module name="RedundantImport">
 			<property name="severity" value="error"/>
 		</module>
+		<module name="UnusedImports">
+			<property name="severity" value="error"/>
+		</module>
```

Comment:
Why is this in your commit? Don't see any mention of this here https://issues.openmrs.org/browse/TRUNK-5056